pkg/operator2/operator.go

```diff
@@ -26,7 +26,7 @@ import (
 )
 
 const (
-	targetName = "openshift-osin" // TODO fix
+	targetName = "openshift-integrated-auth"
```

Comment:
can you outline the new name requirements so we know if this matches

Reply:
from email thread (kube-apiserver, naming conventions from @deads2k  are as follows:
```
ClusterOperator name  "authentication" 
Operator Namespace “openshift-authentication-operator”   -good
Operator Deployment “cluster-authentication-operator”  -good
Operator Git repo “openshift/cluster-authentication-operator”  -will be changed
Operand Namespace “openshift-authentication” -edited  this PR to match
```